### PR TITLE
Adding properties for disabling sort and search on inbox

### PR DESCRIPTION
**Pull request by e-kag**
# Pull Request

* ## A description of the changes proposed in the pull request:
Added ability to disable search and sort in the inbox, through properties: `disable-search` and `disable-sort`

Below are screenshots for these properties:
**None disabled:**
image
**Search disabled:**
image
**Sort disabled**
image
**Search and sort disabled**
image

* ## A reference to a related issue (if applicable):
https://github.com/predixdesignsystem/px-inbox/issues/15

### CODE REVIEW

**Review by MichaelZaporozhets — APPROVED** [review]
clean!

File: px-inbox.html [hunk 1]
```diff
@@ -149,12 +155,26 @@
         readOnly: true,
         computed: '_computeSelectedItemRef(selectedItem, listItems, listItems.*)'
       },
+      /**
+       * Disables search ability.
+       */
+       disableSearch: {
```
Comment (evanjd) [inline px-inbox.html]: Sorry for the pedantry - the props you added are misaligned by one space compared to other props.

**Comment by e-kag** [comment]
@clabot verify

**Review by evanjd — COMMENTED** [review]
The vertical alignment of the search box with sort disabled seems a little off (20px on bottom, 30px on top):

Screen Shot 2019-03-12 at 10 05 15 am
Screen Shot 2019-03-12 at 10 05 07 am

Otherwise it looks good. I'm guessing you'll need to regen lock files to get around the build errors.

File: px-inbox.html (hunk 1 above)
Comment (e-kag) [reply px-inbox.html]: No idea how you even spotted that! Thanks haha

**Review by evanjd — APPROVED** [review]
You can safely ignore the test failure on `travis-ci/push`, our saucelabs token has been revoked.

**Comment by e-kag** [comment]
@evanjd http://gph.is/1VUoo4A

**Pull request merged**